infrastructure/perf/artillery.yml

```
@@ -0,0 +1,14 @@
+config:
+  # target: 'http://localhost:3000'
+  target: "https://staging.khouryofficehours.com"
+  phases:
+    - duration: 60
+      arrivalRate: 30
+scenarios:
+  - flow:
+      - get:
+          url: "/api/v1/profile/entry?userId=1"
```

This was route was changed to 
```suggestion
          url: "/api/v1/login/dev?userId=1"
```